static/app/components/discover/performanceCardTable.tsx

```diff
@@ -0,0 +1,667 @@
+              : cell.includes('NaN')
+              ? cell.replace('NaN', '0.00')
+              : cell}
+          </GridCellNumber>
+        ))}
+      </BodyCellContainer>
+    );
+
+    return rendered;
+  }
+
+  renderResults() {
+    const {isLoading, tableData, releaseTableData, columnOrder, project} = this.props;
+    let cells: React.ReactNode[] = [];
+    let releaseCells: React.ReactNode[] = [];
+
+    if (isLoading) {
+      return cells;
+    }
+    if (!tableData || !tableData.meta || !tableData.data) {
+      return cells;
+    }
+
+    if (!releaseTableData || !releaseTableData.meta || !releaseTableData?.data?.length) {
+      if (FRONTEND_PLATFORMS.includes(project.platform as string)) {
+        const noReleaseTableData = {
+          data: [
+            {
+              user_misery_300: '',
+              p75_measurements_fcp: '',
+              p75_measurements_fid: '',
+              p75_measurements_lcp: '',
+              p75_measurements_cls: '',
+              p75_spans_http: '',
+              p75_spans_db: '',
+              p75_spans_browser: '',
+              p75_spans_resource: '',
+            },
+          ],
+        };
+        releaseTableData!.data = noReleaseTableData.data;
+      } else if (BACKEND_PLATFORMS.includes(project.platform as string)) {
+        const noReleaseTableData = {
+          data: [
+            {
+              user_misery_300: '',
+              apdex_300: '',
+              p75_spans_http: '',
+              p75_spans_db: '',
+            },
+          ],
+        };
+        releaseTableData!.data = noReleaseTableData.data;
+      } else {
+        const noReleaseTableData = {
+          data: [
+            {
+              user_misery_300: '',
+            },
+          ],
+        };
+        releaseTableData!.data = noReleaseTableData.data;
+      }
+    }
+
+    tableData.data.forEach((row, i: number) => {
+      // Another check to appease tsc
+      if (!tableData.meta) {
+        return;
+      }
+      cells = cells.concat(this.renderRow(row, i, columnOrder, tableData.meta));
+    });
+
+    releaseTableData?.data.forEach((row, i: number) => {
+      // Another check to appease tsc
+      if (!releaseTableData?.meta) {
+        return;
+      }
+      releaseCells = releaseCells.concat(
+        this.renderRow(row, i, columnOrder, releaseTableData?.meta)
+      );
+    });
+    return [cells, releaseCells];
+  }
+
+  renderChangeResults() {
+    const {tableData, releaseTableData, project} = this.props;
+    let changeCells: React.ReactNode[] = [];
+
+    if (tableData?.data.length && releaseTableData?.data?.length) {
+      if (FRONTEND_PLATFORMS.includes(project.platform as string)) {
+        const userMiseryChange =
+          +releaseTableData?.data[0].user_misery_300 -
+          tableData?.data[0]['user_misery_300'];
+        const fcpChange =
+          +releaseTableData?.data[0].p75_measurements_fcp -
+          tableData?.data[0]['p75_measurements_fcp'];
```

Comment:
![image](https://user-images.githubusercontent.com/9060071/127315799-27041ecb-8bc9-4d25-843c-9b94be1b8519.png)
I believe in this case the arrow should go up because the number increased in this release. (and it's a bad thing so it should be red)